README.md

```diff
@@ -585,6 +585,28 @@ Actions supported:
 * `client.reserved_ip_actions.assign(ip: reserved_ip.ip, droplet_id: droplet.id)`
 * `client.reserved_ip_actions.unassign(ip: reserved_ip.ip)`
 
+## Reserved IPv6 resource
+
+    client = DropletKit::Client.new(access_token: 'TOKEN')
+    client.reserved_ipv6s #=> DropletKit::ReservedIpv6Resource
```

Comment:
I know that not all of the resources have examples in this section, but it's definitely a nice-to-have, so lets include it here:
```suggestion
    client.reserved_ipv6s #=> DropletKit::ReservedIpv6Resource
    reserved_ipv6 = DropletKit::ReservedIpv6.new(
      region_slug: 'nyc1'
    )
```